### PR TITLE
Fix procedural pipeline on dev 

### PR DESCRIPTION
**Pull request by uq20042**

### CODE REVIEW

**Comment by James-Millan** [comment]
will be branching off here for the meantime, need alexs approval or to fix the networking stuff then we'll merge to dev

**Comment by alexelwood14** [comment]
Merge the PR for fix-dev-netcode into this branch before merging this PR. I will approve this PR once that has happened.

**Review by alexelwood14 — CHANGES_REQUESTED** [review]
See previous comment

**Review by James-Millan — APPROVED** [review]

**Review by alexelwood14 — APPROVED** [review]

**Pull request merged**
